packages/react/src/components/Avatar/Avatar.tsx

```diff
@@ -14,6 +16,12 @@ import {
 } from '../../lib'
 
 export interface AvatarProps extends UIComponentProps {
+  /**
+   * Accessibility behavior if overridden by the user.
+   * @default defaultBehavior
+   */
+  accessibility?: Accessibility
```

Comment:
Can we add this in interface (or if it should be use in all UI components, maybe even move to the UIComponentProps)? At least common interface would allow us to reuse the same description, although not sure how we can add different @defaults there..

Reply:
Btw will we still need these @default description, after removing `react-docgen-typescript`?

Reply:
I'm note sure that the issues with `@default` will be solved right now while some components have different defaults 🐱